ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -138,11 +139,17 @@ private HttpServerOptions applyConfigTlsSettingsTo(final HttpServerOptions input
       final String password = readSecretFromFile(tlsConfig.getKeyStorePasswordFile().toPath());
```

Comment:
Would it be possible to write a unit test to show that the config gets applied correctly to the httpServerOptions? Not sure how much Vertx will get in the way.

Reply:
Yes, we need it - think a better answer is to refactor this now that its getting heavier, and unit test the broken out class